strings/strings.json

```diff
@@ -1578,6 +1578,11 @@
     "zh-cn": "没有找到与 {0} 相关的结果",
     "zh-tw": "沒有找到與 {0} 相關的結果"
   },
+  "noResult": {
+    "en": "No results",
+    "fr": "Aucun résultat",
```

Comment:
Should be `Aucun résultats` (plural) to match the other ones.

Reply:
http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=1440 😄

Reply:
After all these years of french classes, I still don't know how to write 😅

Reply:
🤣🤣🤣